docs/deploy-development-rp.md

```diff
@@ -104,9 +104,19 @@ mock a cluster MSI. This script will also create the platform identities, platfo
 
 1. Run the RP
 
-   ```bash
-   make runlocal-rp
-   ```
+    Option 1: using local compilation and binaries (requires local `go`/build dependencies/etc):
+    ```bash
+    make runlocal-rp
+    ```
+
+    Option 2: using containerized build and run (requires local `podman` and `openvpn`):
+    ```bash
+    # establish a VPN connection to the shared dev environment Hive cluster
+    sudo openvpn secrets/vpn-${LOCATION}.ovpn &
```

Comment:
From my testing it must be `vpn-aks-$LOCATION.ovpn`

Reply:
There should be variations of both `secrets/vpn-${LOCATION}.ovpn` and `secrets/vpn-aks-${LOCATION}.ovpn` for every dev region. I've always used the non-aks one locally :thinking: and have tested this in `westeurope` with success... do we have diverging dev environments maybe?

Reply:
@ehvs Can you follow up on this documentation task in a new PR? Maybe we can reconcile the difference, but I don't think we should continue to block the PR.